src/test/java/org/folio/rest/HeadersForwardingTest.java

```diff
@@ -98,10 +99,23 @@ public void testPostBlUsersLogin() {
 
 
     WireMock.stubFor(
-      WireMock.get("/users?query=username=" + USERNAME)
+      WireMock.get("/users?query=username==" + USERNAME)
         .willReturn(WireMock.okJson(users.encode()))
     );
 
+    String incorrectUserName = USERNAME + "_one";
+    JsonObject incorrectUser = new JsonObject()
+      .put("username", incorrectUserName)
+      .put("id", UUID.randomUUID().toString());
+
+    WireMock.stubFor(
+      WireMock.get("/users?query=username==" + incorrectUserName)
```

Comment:
Seems like this stub isn't used in the test. Given we don't emulate user storage in tests, I think it isn't possible to verify this behavior correctly. The only thing we can do with WireMock is to verify that a call was made to mod-users with `==` in CQL parameter.

Reply:
Fix